introduction/installation/python-bindings.md

```diff
@@ -6,4 +6,269 @@ parent: Installation
 grand_parent: Introduction / Getting Started
 ---
 
-## Serverless
+# Python Bindings
+
+GlareDB can be used in Python environments along with [Pandas] and [Polars] to
+work with distributed data.
+
+For more information, refer to our in-depth [blog post Working with Python].
+
+## Install
+
+The `glaredb` module is available on PyPI <https://pypi.org/project/glaredb/>.
+To install the module:
+
+```console
+pip install glaredb
+```
+
+## Usage
+
+### Embedded Connection
+Once installed, GlareDB can be used inside your python scripts by importing the
+`glaredb` library, and creating a connection using `connect`.
+
+```python
+import glaredb
+
+con = glaredb.connect()
+con.sql("select 1").show()
+```
+
+By default, `connect` will use an in-memory database. A path may be provided to
+`connect` to persist data after the python script exits.
```

Comment:
```
A path may be provided to `connect` to persist data after the python script exits.
```

The end of the sentence ("after the python scripts exits") feels potentially misleading, I think we can change the sentence to (borrowed from Node.js doc)

```
To persist data locally, specify a location:
```

or similar